build.sbt

```diff
@@ -37,6 +37,7 @@ inThisBuild(Seq(
   // https://www.scala-sbt.org/1.x/docs/Cached-Resolution.html
   updateOptions := updateOptions.value.withCachedResolution(true),
   resolvers ++= Seq(Resolver.sonatypeRepo("releases")), // libraries that haven't yet synced to maven central
+  scalacOptions += "-Ypartial-unification",
```

Comment:
this is moved to the top level.  It was already in stripe-intent and payment-api.  It is needed to do sequence on Either[String, A] without needing a type alias. 

> Cats relies on improved type inference via the fix for SI-2712, which is not enabled by default.

from https://typelevel.org/cats/